### PR TITLE
Annotation of PXD010708 and PXD000790

### PR DESCRIPTION
**Pull request by akiena**

### CODE REVIEW

**Comment by ypriverol** [comment]
thanks @akiena for contributing. The dataset PXD000790 is based on a cell line (murine myoblast cell line) you should try to look for the cellline information and add that info into the annotations. The phenotype -> Sample 1 is not correct. 

Definition of the phenotype by OLS -> `The observable form taken by some character (or group of characters) in an individual or an organism, excluding pathology and disease. The detectable outward manifestations of a specific genotype.`

File: annotated-projects/PXD000790/sdrf.tsv [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
+Sample1	Mus musculus	Cell culture	"NT=C2C12;AC=EFO:0001098"	unkown	Run 1	QEII12347.raw	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2015/05/PXD000790/QEII12347.raw	1	"AC=MS:1002038;NT=label free sample"	"NT=Trypsin;AC=MS:1001251"	"NT=LTQ Orbitrap;AC=MS:1000449"	CID	5 ppm	0.4 Da	"NT=Oxidation; MT=variable; PP=Anywhere; AC=Unimod:35; TA=M; MM=15.994915"	"NT=Acetyl; MT=variable; PP=Protein N-Term; AC=Unimod:1; TA=.; MM=42.010567"	"NT=  Gln->pyro-Glu;  MT=variable; PP=Protein N-Term; AC=Unimod:28; TA=.; MM=-17.0305"	"NT=  Carbamidomethyl;  MT=variable; PP=Protein C-Term; AC=Unimod:4; TA=.; MM=57.0513"	"NT=  Glu->pyro-Glu  MT=variable; PP=Protein N-Term; AC=Unimod:27; TA=.; MM=  -18.0153"	"NT=    Dehydration  MT=variable; PP=Protein C-Term; AC=Unimod:23; TA=.; MM=  -18.0153"	wildtype	"NT=undifferentiated;AC=PATO:0002100"
```
Comment (ypriverol) [inline annotated-projects/PXD000790/sdrf.tsv:2]: The disease should be `unavailable` not unknown

Comment (ypriverol) [reply annotated-projects/PXD000790/sdrf.tsv:2]: `undifferentiated` do not exits as factor value

**Comment by akiena** [comment]
hey @ypriverol which changes do you want in this project ? Could I do something else ? Thank you

**Pull request merged**